clang/lib/Driver/ToolChains/ClassicFlang.cpp

```diff
@@ -355,6 +355,9 @@ void ClassicFlang::ConstructJob(Compilation &C, const JobAction &JA,
       CommonCmdArgs.push_back("0x1000000");
     else if (GDwarfArg->getOption().matches(options::OPT_gdwarf_5)) // -gdwarf-5
       CommonCmdArgs.push_back("0x2000000");
+    else if (GDwarfArg->getOption().matches(
+                 options::OPT_gpubnames)) // -gpubnames
+      CommonCmdArgs.push_back("0x40000000");
```

Comment:
@SouraVX , thanks for adding me as a reviewer. The only thing I cannot figure out is where the value of `0x4000000` comes from? 
Perhaps I am looking in a wrong place, but the only place where I found the dwarf flags with similar values to these [is here](https://github.com/flang-compiler/flang/blob/master/tools/flang2/docs/xflag.n#L3297) and the value of `040000000` is described as "Do not generate include file tables". Can you point me to the code or documentation of this flag value, please?

Reply:
Thanks @michalpasztamobica for reviewing this. This PR is pretty old and hence might be missing pieces(Apologies for that) . In this PR intent is to use/bind an available bit to this option more details: https://github.com/flang-compiler/flang/blob/master/tools/flang2/docs/xflag.n#L3306 bit. 
This piece of diff is missing from https://github.com/flang-compiler/flang/pull/934, I'll update the PR soon.
```
diff --git a/tools/flang2/docs/xflag.n b/tools/flang2/docs/xflag.n
index 2ad1b02e..eddf910b 100644
--- a/tools/flang2/docs/xflag.n
+++ b/tools/flang2/docs/xflag.n
@@ -3254,7 +3254,7 @@ Generating eh_frame.
 .XB 0x20000000:
 Generating eh_frame with .cfi directives: requires 120,0x10000000 to be on
 .XB 0x40000000
-AVAILABLE
+Generate .debug_names/.debug_pubnames section.
 .XB 0x80000000:
 no license check in executable.
```

Reply:
Modifications/rebase Done @michalpasztamobica . Now you can apply these 2 PR in order and test :)
Order: Apply this one first -> then 934. Build and test.